pyproject.toml

```diff
@@ -106,6 +108,19 @@ full = [
     "defusedxml",
     "neo",
 ]
+full = [
+    "mne[full-no-qt]",
+    "PyQt6!=6.6.0",
+    "PyQt6-Qt6!=6.6.0,!=6.7.0",
+]
+full-pyqt6 = [
+    "mne[full]"
+]
+full-pyside6 = [
+    "mne[full-no-qt]",
+    "PySide6"
+]
```

Comment:
Actually … we could also do:
```python
pyqt6 = []
pyside6 = [
    "PySide6"
]
```
and then the user could install it by selecting `mne[full-no-qt,pyside6]` … or would that be awkward?